.travis/before_install.sh

```diff
@@ -8,6 +8,9 @@ if [ ! -n "$NUMCORES" ]; then
 fi
 echo Using $NUMCORES cores
 
+echo ARST
```

Comment:
I'll clean this up before pushing (and below), but I don't want to retrigger all the tests right now